safe-global/web-core
# fix: switching hardware wallet networks

## What it solves

Resolves #1073

## How this PR fixes it

If a hardware wallet is connected, it is diconnected and reconnected when switching network as it is not possible to change the network of a hardware wallet.

## How to test it

Open the Safe on a network where a Ledger/Trezor is enabled and connect it. Switch to another network that also supports Ledger/Trezor. Observe the address scanner modal open.
